toolkit/analysis_and_visualization/classification/assets/styles/simplePagination.css

```diff
@@ -0,0 +1,282 @@
+  cursor: pointer;
+}
+
+.ellipse input {
+  width: 3em;
+}
+
+/* ------------------------------------*\
+	Compact Theme Styles
+\*------------------------------------ */
+.compact-theme span {
+  cursor: pointer;
+}
+
+.compact-theme a,
+.compact-theme span {
+  float: left;
+  color: #333;
+  font-size: 14px;
+  line-height: 24px;
+  font-weight: normal;
+  text-align: center;
+  border: 1px solid #aaa;
+  border-left: none;
+  min-width: 14px;
+  padding: 0 7px;
+  box-shadow: 2px 2px 2px rgba(0, 0, 0, 0.2);
+  background: #efefef;
+
+  /* Old browsers */
+  background: -moz-linear-gradient(top, #fff 0%, #efefef 100%);
+
+  /* FF3.6+ */
+  background: -webkit-gradient(linear, left top, left bottom, color-stop(0%, #fff), color-stop(100%, #efefef));
+
+  /* Chrome,Safari4+ */
+  background: -webkit-linear-gradient(top, #fff 0%, #efefef 100%);
+
+  /* Chrome10+,Safari5.1+ */
+  background: -o-linear-gradient(top, #fff 0%, #efefef 100%);
+
+  /* Opera11.10+ */
+  background: -ms-linear-gradient(top, #fff 0%, #efefef 100%);
+
+  /* IE10+ */
+  background: linear-gradient(top, #fff 0%, #efefef 100%);
+
+  /* W3C */
+}
+
+.compact-theme a:hover,
+.compact-theme li:not(.disabled):not(.active) span:hover {
+  text-decoration: none;
+  background: #efefef;
+
+  /* Old browsers */
+  background: -moz-linear-gradient(top, #efefef 0%, #bbb 100%);
+
+  /* FF3.6+ */
+  background: -webkit-gradient(linear, left top, left bottom, color-stop(0%, #efefef), color-stop(100%, #bbb));
+
+  /* Chrome,Safari4+ */
+  background: -webkit-linear-gradient(top, #efefef 0%, #bbb 100%);
+
+  /* Chrome10+,Safari5.1+ */
+  background: -o-linear-gradient(top, #efefef 0%, #bbb 100%);
+
+  /* Opera11.10+ */
+  background: -ms-linear-gradient(top, #efefef 0%, #bbb 100%);
+
+  /* IE10+ */
+  background: linear-gradient(top, #efefef 0%, #bbb 100%);
+
+  /* W3C */
+}
+
+.compact-theme li:first-child a,
+.compact-theme li:first-child span {
+  border-left: 1px solid #aaa;
+  border-radius: 3px 0 0 3px;
+}
+
+.compact-theme li:last-child a,
+.compact-theme li:last-child span {
+  border-radius: 0 3px 3px 0;
+}
+
+.compact-theme .current {
+  background: #bbb;
+
+  /* Old browsers */
+  background: -moz-linear-gradient(top, #bbb 0%, #efefef 100%);
+
+  /* FF3.6+ */
+  background: -webkit-gradient(linear, left top, left bottom, color-stop(0%, #bbb), color-stop(100%, #efefef));
+
+  /* Chrome,Safari4+ */
+  background: -webkit-linear-gradient(top, #bbb 0%, #efefef 100%);
+
+  /* Chrome10+,Safari5.1+ */
+  background: -o-linear-gradient(top, #bbb 0%, #efefef 100%);
+
+  /* Opera11.10+ */
+  background: -ms-linear-gradient(top, #bbb 0%, #efefef 100%);
+
+  /* IE10+ */
+  background: linear-gradient(top, #bbb 0%, #efefef 100%);
+
+  /* W3C */
+  cursor: default;
+}
+
+.compact-theme .ellipse {
+  background: #eaeaea;
+  padding: 0 10px;
+  cursor: default;
+}
+
+/* ------------------------------------*\
+	Light Theme Styles
+\*------------------------------------ */
+.light-theme span {
```

Comment:
Codacy found an issue: [Expected selector ".light-theme span" to come before selector ".compact-theme li:not(.disabled):not(.active) span:hover" (no-descending-specificity)](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6397592)